src/containers/Tenant/Diagnostics/Consumers/Consumers.tsx

```diff
@@ -0,0 +1,90 @@
+import {useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import block from 'bem-cn-lite';
+
+import DataTable, {Column} from '@yandex-cloud/react-data-table';
+
+import {DEFAULT_TABLE_SETTINGS} from '../../../../utils/constants';
+import {useAutofetcher} from '../../../../utils/hooks';
+import {Search} from '../../../../components/Search';
+import {getDescribe} from '../../../../store/reducers/describe';
+
+import i18n from './i18n';
+
+import './Consumers.scss';
+
+const b = block('ydb-consumers');
+
+interface Consumer {
+    name: string;
+}
+
+interface ConsumersProps {
+    path: string;
+}
+
+export const Consumers = ({path}: ConsumersProps) => {
+    const dispath = useDispatch();
+
+    const fetchData = () => {
+        dispath(getDescribe({path}));
+    };
+
+    useAutofetcher(fetchData, [path]);
+
+    const data = useSelector((state: any) => state.describe.data);
+    const consumersNames: string[] =
+        data[path]?.PathDescription?.PersQueueGroup.PQTabletConfig.ReadRules || [];
+
+    const consumers: Consumer[] = consumersNames.map((name: string) => ({name}));
+
+    const [consumersToRender, setConsumersToRender] = useState<Consumer[]>(consumers);
+
+    useEffect(() => {
+        setConsumersToRender(consumers);
+    }, [consumersNames]);
```

Comment:
better memoize `consumerNames` and use it as a dependency here
but if the transformation logic moves to a selector, this becomes even cleaner